src/main/java/mate/academy/book/BookDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+package mate.academy.book;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.connection.ConnectionUtils;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+
+    @Override
+    public Book create(Book book) {
```

Comment:
In the `create` method, ensure that the SQL statement uses the correct table name `books` as specified in the task description. The current statement uses `book`, which may lead to errors if the table name is incorrect.